hr_timesheet_role/report/report_timesheet_templates.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
     <!--
-        Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+        Copyright 2018-2019 Brainbean Apps (https://brainbeanapps.com)
         License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
     -->
 
     <template id="report_timesheet" inherit_id="hr_timesheet.report_timesheet">
-        <xpath expr="//th/span[text()='Responsible']/.." position="after">
+        <xpath expr="(//th)[count(//td/span[@t-field='l.user_id.partner_id.name']/../preceding-sibling::*)+1]" position="after">
```

Comment:
@pedrobaeza "After column whose data is `partner_id.name`" - so no way to make this one simplier